grammar.js

```diff
@@ -232,7 +232,8 @@ module.exports = grammar({
     ),
 
     parameter_declaration: $ => seq(
-      field('name', commaSep($.identifier)),
+      field('name', $._identifier),
+      repeat(seq(',', field('name', $._identifier))),
```

Comment:
I think that you could still use the `commaSep` function for this:

```js
commaSep(field('name', $.identifier)),
```

Reply:
Of course; done.